src/meson.build

```diff
@@ -3,20 +3,27 @@
 phosphor_dbus_interfaces = dependency('phosphor-dbus-interfaces')
 phosphor_logging = dependency('phosphor-logging')
 sdbusplus = dependency('sdbusplus')
+nlohmann_json_dep = dependency('nlohmann_json')
 
 rbmc_data_sync_sources = [
-    'rbmc_data_sync_main.cpp'
+    files(
+        'data_sync_config.cpp',
+        'manager.cpp'
```

Comment:
I noticed this file is missing from this commit but was added in a later commit. Please fix this. 

I hope you ran the local CI for each commit. (I know, working with multiple commit is pain :) )